scripts/configure-hypervisor.sh

```diff
@@ -217,6 +217,11 @@ generate_qemu_options() {
 	qemu_options+=(size:--disable-vnc-png)
 	qemu_options+=(size:--disable-vnc-sasl)
 
+	# Disable PAM authentication, as it's a feature used
+	# together with VNC access, that's also disabled.
+	# See QEMU commit 8953caf for more details
+	qemu_options+=(size:--disable-auth-pam)
```

Comment:
snap CI is not happy, This option was added in QEMU 4, can you add validate it?  i.e `if [[ "${qemu_version_major}" -ge 4 `

```
ERROR: unknown option --disable-auth-pam
Try './configure --help' for more information
````

Reply:
Thanks @devimc , I have updated it.